recipes/Momo.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Momo_Recipe</title>
+</head>
+<body>
+    <h1>Momo</h1>
+    <img src="Momo-Img.jpg" style = "width:70%">
+    <p>Momo is made from wheat flour, vegetable oil, chopped onion, garlic, sesame, green chilies, tomatoes, meat (preferably beef), mustard powder, ginger juice and a blend of Nepalese herbal spices. 
+    </p>
```

Comment:
formatting in this & other html files looks off. As per my earlier comment, configure Prettier extension to format your files on Save